### ISSUE TITLE
[BUG] regression of fix #127

### ISSUE THREAD

**Issue by slashdd** [soscleaner/soscleaner#143]
I didn't expect 7b2422981b7d577e9c85086ece07b69ea780e942 (when I reported the RFE) by switching from WARNING to INFO to make the disclaimer disappear from STDOUT.

I believe the disclaimer MUST be clearly seen by user, but not mark as a WARNING.

With that being said, after all maybe CONSOLE is our best option to avoid the WARNING and to still be seen by user in STDOUT ?

**Comment by slashdd** [comment]
Current output running soscleaner from master HEAD:
```
07-05 16:54:52 soscleaner CONSOLE: Log File Created at /tmp/soscleaner-2371308501901533.log
07-05 16:54:52 soscleaner CONSOLE: Creating Loopback Network Entry
07-05 16:54:52 soscleaner CONSOLE: Beginning SOSReport Extraction
```

Expected output:
```
07-05 16:47:56 soscleaner CONSOLE: Log File Created at /tmp/soscleaner-1235416098963411.log
07-05 16:47:56 soscleaner CONSOLE: soscleaner is a tool to help obfuscate sensitive information from an existing sosreport.
07-05 16:47:56 soscleaner CONSOLE: Please review the content before passing it along to any third party.
```

**Comment by slashdd** [comment]
Fixed ee92b212ed05851b9906a3c2d5f1cadf862face6
